shared/middleware/helpers/storage_apps.rb

```diff
@@ -63,6 +64,42 @@ def update(channel_id, value, ip_address)
     JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id, updatedAt: row[:updated_at])
   end
 
+  def get_abuse(channel_id)
+    owner, id = storage_decrypt_channel_id(channel_id)
+
+    row = @table.where(id: id).exclude(state: 'deleted').first
+    raise NotFound, "channel `#{channel_id}` not found" unless row
+
+    row[:abuse_score]
+  end
+
+  def increment_abuse(channel_id)
+    owner, id = storage_decrypt_channel_id(channel_id)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `owner`. Use `_` or `_owner` as a variable name to indicate that it won't be used.